js/ad_manager_vast.js

```diff
@@ -640,6 +884,52 @@ OO.Ads.manager(function(_, $) {
     };
 
     /**
+    * This method pings all the ad's error URLs with a specific error code if the error URL
+    * contains the macro, "[ERRORCODE]". If ad has a parent wrapper, then go up the chain and ping
+    * wrapper's error urls as well.
+    * @public
+    * @method Vast#trackError
+    * @param {number} code Error code
+    * @param {boolean} currentAdId Ad ID of current ad
+    */
+    this.trackError = _.bind(function trackErrorHelper(code, currentAdId) {
+      if (currentAdId && currentAdId in this.errorInfo) {
+        pingURLs(this.errorInfo[currentAdId].errorUrls);
+        var parentId = this.errorInfo[currentAdId].wrapperParentId;
+
+        // ping parent wrapper's error urls too if ad had parent
+        if (parentId) {
+          this.trackErrorHelper(code, parentId);
```

Comment:
Hm - does the function need two names?  Does trackError not map exactly to trackErrorHelper except that the inner one is not bound to this?